src/main/java/de/bonndan/nivio/util/SafeAssign.java

```diff
@@ -13,7 +14,7 @@ public class SafeAssign {
      * @param c consumer
      */
     public static void assignSafe(String s, Consumer<String> c) {
-        if (s != null) c.accept(s);
+        c.accept(Objects.requireNonNullElse(s, ""));
```

Comment:
Doesn't this change the behavior?

Before, `accept()` was not called when `null` is provided.

Reply:
Yes we want to change the behaviour. It should save an empty string now instead of doing nothing if null is provided. Removing items from yaml completely had the side effect that the data didn't get removed in nivio. It was missleading.